### ISSUE TITLE
Status within UN

### ISSUE THREAD

**Issue by TomTomRixRix** [model-united-nations-of-luebeck/CMS-backend#2]
At the moment I have the following status' for member organisations that are simulated at MUNs:
```
MEMBER_STATE
OBSERVER_STATE
FORMER_MEMBER
NON_GOVERNMENTAL_ORGANIZATION 
INTER_GOVERNMENTAL_ORGANIZATION 
UN_SUB_BODY 
```

Are there other roles an institution might have when it is member of a UN forum?

**Comment by LeoRoe** [comment]
Those are all I had in mind and got in contact with yet. I just did some research and it appears that INGOs/IGOs either are considered with observer rights/status or consultative status. But as we directly name them as NGOs/IGOs (which makes sense) there is no need for differentiation.
There are two things to clarify:
1. Organisations such as the African Union, the European Union or the Carribean Community are considered as IGOs according to the current model, if I got it right.
2. According to Wikipedia (with other sources) there are some entities, which are not states and maybe would not be considered as NGOs, but have observer status, such as the Sovereign Military Order of Malta. Which category could it belong to or do we need a new one? Is it likely that it will ever be of use? Or should we pool some and simplify things?

**Comment by TomTomRixRix** [comment]
Perhaps it might make sense to think about where we will use this information about the status of a member organisation in our CMS. At the moment we only use it to create different placards (which are coloured) in order to determine if a member organisation has voting rights. Thus, we only have two groups: Voting members (member states) and non-voting members (all other organisations). However, in the future we might want to make a distinction somewhere (e.g. NGOs get different colours than IGOs or states are treated in another manner than organisations. Therefore it might make sense to model the difference status in the database already.

Regarding the two points:
1. Yes, I would according to [this Wikipedia article](https://en.wikipedia.org/wiki/United_Nations_General_Assembly_observers) put them into the IGO category. At least for the EU that's only partly correct as it also has some supranational powers as stated in the Wikipedia article, but I think the intergovernmental aspects are more important in this case.

2. That is indeed a tricky question. We could make a new category for other observer entities. Personally, I would have assigned them into the NGO category, because they are neither states nor UN sub-bodies or IGOs. But I don't know what definition of NGO we apply here. For me, all entities and organisations that are not related to governments (thus not intergovernmental) are NGOs. So NGOs is more or less the pooling category for me. Despite the fact that we will probably never simulate such an organisation it is no extra effort to create a new pooling category so it might make sense to simply to so now in order to make decisions easier in the future.

Perhaps it is a good idea to write some documentation and explanation for these categories somewhere (either source code, help texts or in the reference document) so that we and others will understand these categories in the future.

**Comment by LeoRoe** [comment]
A further thing the status could be useful for: Organzizing the list of opening speeches, as they are (as far as I know) optional for NGOs/IGOs/Observers and not possible for former members (not sure about that). But yes, the distinction between NGO and IGO seems to be only for statistic purposes at the moment or to model the reality as semantically correct as possible.

Yes, probably it would be best to widen the definition of "NGO", as there are only very few observers, which would not exactly fit into the definition (according to this not well cited `[Wikipedia article section](https://en.wikipedia.org/wiki/International_Committee_of_the_Red_Cross#Legal_status)` ) and document it.

**Comment by TomTomRixRix** [comment]
Indeed, at least in the future there might be a differentiation between the various member organisation categories at the opening speeches.

The ICRC legal status is really interesting, I didn't know that before and always thought it was just another NGO. But for our purposes it's probably best to simply put it into the NGO category although it's only right to some extend.
